### ISSUE TITLE
Confused about two files.

### ISSUE THREAD

**Issue by ddfssdfkofdsfighdfuigf** [nwjs/nw.js#4799]
nw_100_percent.pak
nw_200_percent.pak

What exactly do these contain? I'm looking at the 64-bit Windows version of v0.14.3.

Bonus questions: what does nw.dll contain and why is it 92 MB? Also, what do all the other .dlls and other files do? I'd like them neatly listed and explained if possible, thanks.

**Comment by rogerwang** [comment]
nw_200_percent.pak is for high DPI screens. Please submit questions to our mailing list: https://groups.google.com/forum/#!forum/nwjs-general

**Comment by ddfssdfkofdsfighdfuigf** [comment]
I don't understand your answer.

Also, I cannot send any e-mails. Why not answer here?

**Comment by iteufel** [comment]
@ddfssdfkofdsfighdfuigf  Have you read this:

Read the following before submitting your issue, thank you.

* Please use English so all users can read, or the issue will be closed directly.
* Use our mailing list to ask questions which don't imply bug or feature request: https://groups.google.com/forum/#!forum/nwjs-general
* State which version are you using
* State which OS version are you running with
* Provide a reproducible case if possible
* Attach the crash dump file if it's a crash: http://docs.nwjs.io/en/latest/For%20Developers/Understanding%20Crash%20Dump/#find-minidump-file

Issues meet the checklist above will be processed before those don't.

**Comment by frankhale** [comment]
> Bonus questions: what does nw.dll contain and why is it 92 MB? 

It's 92 megabytes because it contains all of the awesome stuff found in the Chromium Content layer. 

> Also, what do all the other .dlls and other files do?

They are additional infrastructure either needed directly by the Chromium Content layer or additional support provided by NW.js (for instance Node.dll).
